Code/Tools/ProjectManager/Source/NewProjectSettingsScreen.cpp

```diff
@@ -130,6 +130,29 @@ namespace O3DE::ProjectManager
         {
             m_templates = templatesResult.GetValue();
 
+            // Add in remote templates
+            auto remoteTemplatesResult = PythonBindingsInterface::Get()->GetProjectTemplatesForAllRepos();
+            if (remoteTemplatesResult.IsSuccess() && !remoteTemplatesResult.GetValue().isEmpty())
+            {
+                QVector<ProjectTemplateInfo>& remoteTemplates = remoteTemplatesResult.GetValue();
+                for (ProjectTemplateInfo& remoteTemplate : remoteTemplates)
+                {
+                    auto rem = AZStd::find_if(
+                        m_templates.begin(),
+                        m_templates.end(),
+                        [remoteTemplate](const ProjectTemplateInfo& value)
+                        {
+                            return remoteTemplate.m_name == value.m_name;
+                        });
+                    if (rem == m_templates.end())
+                    {
+                        m_templates.append(remoteTemplate);
+                    }
+                }
```

Comment:
can we const some of this and rename the `rem` variable to something that represents better what `find_if` returns? like `iter` or `found`
```suggestion
                for (const ProjectTemplateInfo& remoteTemplate : remoteTemplates)
                {
                    const auto found = AZStd::find_if( m_templates.begin(), m_templates.end(),
                        [&remoteTemplate](const ProjectTemplateInfo& value)
                        {
                            return remoteTemplate.m_name == value.m_name;
                        });
                    if (found == m_templates.end())
                    {
                        m_templates.append(remoteTemplate);
                    }
                }
```

Reply:
Also this can use `AZStd::ranges::find_if` as well to supply a range container instead of two iterators.